core/src/test/resources/HistoryServerExpectations/one_app_multi_attempt_json_expectation.json

```diff
@@ -11,6 +14,9 @@
     "completed" : true
   }, {
     "attemptId" : "1",
+    "startTimeEpoch" : 1426533910242,
+    "endTimeEpoch" : 1426533945177,
+    "lastUpdatedEpoch" : "",
```

Comment:
I'll detail it better just to make sure we're on the same page: HistoryServerSuite looks at a test json and before comparing it to the exception json file (this file) checks if ```lastUpdated``` (and now also ```lastUpdatedEpoch```) exists, if so it over writes it's value with an empty string. I'm not sure why they decide to do it the sway but it is detailed in a comment on line 157 in HistoryServerSuite.scala